cpg_workflows/jobs/validation.py

```diff
@@ -223,20 +215,15 @@ def parse_and_post_results(
             for sub_key, sub_value in SUMMARY_KEYS.items():
                 summary_data[f'{summary_key}::{sub_value}'] = str(line[sub_key])
 
-    with out_file.open('w', encoding='utf-8') as handle:
+    with to_path(out_file).open('w', encoding='utf-8') as handle:
         json.dump(summary_data, handle)
 
-    # post results to metamist
-    AnalysisApi().create_new_analysis(
-        project=get_config()['workflow']['dataset'],
-        analysis_model=AnalysisModel(
-            sample_ids=[sample.id],
-            type=AnalysisType('qc'),
-            status=AnalysisStatus('completed'),
-            output=str(happy_csv.parent),
-            meta=summary_data,
-            active=True,
-        ),
+    # NOTE: This change is untested.
+    get_metamist().create_analysis(
```

Comment:
@vivbak just checking, is this new method version already available?

Reply:
Do you mean the updated create_analysis endpoint in metamist?

Reply:
Yup! AFAIK this is a renamed method in the `metamist` library, so this is currently available 👌